go-controller/pkg/ovn/gateway_init.go

```diff
@@ -140,6 +141,15 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 			kapi.ProtocolUDP:  k8sNSLbUDP,
 			kapi.ProtocolSCTP: k8sNSLbSCTP,
 		}
+		workerK8sNSLbTCP, workerK8sNSLbUDP, workerK8sNSLbSCTP, err := loadbalancer.GetWorkerLoadBalancers(nodeName)
+		workerProtoLBMap := map[kapi.Protocol]string{
+			kapi.ProtocolTCP:  workerK8sNSLbTCP,
+			kapi.ProtocolUDP:  workerK8sNSLbUDP,
+			kapi.ProtocolSCTP: workerK8sNSLbSCTP,
+		}
```

Comment:
this should go after the `if err=!nil` block, if the operation fails you return and there is no need to initalize this variable

Reply:
yep, thanks.